build.gradle

```diff
@@ -41,7 +41,7 @@ apply plugin: LocalizationPlugin
 apply from: 'eclipse.gradle'
 
 group = "org.jabref"
-version = "5.0.0"
+version = "5.0.1"
```

Comment:
I think, `5.0.0.1` is sufficient (it just needs to be bigger than `5.0.0.0` for the installer to upgrade automatically).

Reply:
needs to be a three digit number. :/

https://docs.microsoft.com/de-de/windows/win32/msi/productversion?redirectedfrom=MSDNhttps://docs.microsoft.com/de-de/windows/win32/msi/productversion?redirectedfrom=MSDN

Reply:
I did not notice that we have a new thing here. In 5.0 alpha, we could just put `5.0-alpha` here. See 0c342bdc74a2cea5ebb768e370c121829340ca8e. Back in the install4j-times, we just had a different version number `threeDotVersion`, which we handled separatetly. The enabled us to cope with a) custom JabRef versioning (which is not semantic versioning) and b) versioning required by external tooling.

Reply:
5.0 alpha didn't use jpackage. As @LinusDietz said, the WiX toolbox / MSI installer needs a clean three digit number.